build-support/bin/check_pants_pex_abi.sh

```diff
@@ -0,0 +1,77 @@
+#!/usr/bin/env bash
```

Comment:
Presumably this script was helpful when you were debugging this PR. You're also sure it's worth its weight going forward?

Reply:
This script is meant above all to act as an integration test / check for regressions. The most likely scenario where it will catch us from a regression is that we change how we handle interpreter constraints or resolve `$PY`, and that that change results in this no longer working as expected. Other scenarios could be changes made to `.travis.yml` or the Dockerfile breaking things. 

This is why I productionized it. The original was an inline command in `.travis.yml` with a series of `&&` and `;` statements, and then I realized this would be useful to have permanently as a sanity check and that we can even use it to verify `abi3` is setup correctly when building Py3 wheels.

At the cost of this safety / regression test, we have more code and about 1 extra second to each of the wheel building shards. Output from running script with `time` on my mac:
```bash
$ time ./build-support/bin/check_pants_pex_abi.sh cp27m
0.47s user 0.94s system 93% cpu 1.508 total
```

Fine with deleting it if you all prefer but I think we get more benefit than harm here.

Reply:
Given the above I am in favor of keeping this script until we have a test that exercises actually resolving each of the built wheels and running them on a python interpreter with the given abi.